pallas-primitives/src/alonzo/model.rs

```diff
@@ -612,7 +610,7 @@ pub struct MintedTx<'b> {
     pub success: bool,
 
     #[n(3)]
-    pub auxiliary_data: Nullable<KeepRaw<'b, AuxiliaryData>>,
+    pub auxiliary_data: KeepRaw<'b, Option<AuxiliaryData>>,
 }
```

Comment:
This was a tough one, since if I replace `Nullable` by `Option` in place, it turns into a "optional field"  which is not encoded at all if nil.

I opted for having the `Option` inside `KeepRaw` in order to preserve the cbor.